examples/dataset/classification/classification/binary/upload_dataset.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021-2024 Kolena Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict
+
+import pandas as pd
+from classification.binary.constants import BUCKET
+from classification.binary.constants import DATASET
+
+import kolena
+from kolena.dataset import register_dataset
+from kolena.workflow._datatypes import _get_full_type
+from kolena.workflow._datatypes import _serialize_dataobject
+from kolena.workflow.annotation import ClassificationLabel
+from kolena.workflow.io import _dataframe_object_serde
+
+
+def to_serialized_dataframe(df: pd.DataFrame, column: str) -> pd.DataFrame:
+    result = _dataframe_object_serde(df, _serialize_dataobject)
+    result[column] = result.to_dict("records")
+    return result[[column]]
+
+
+def to_label_object(x: str) -> Dict[str, str]:
+    label = ClassificationLabel(x)
+    return {
+        "label": label.label,
+        "data_type": _get_full_type(label),
```

Comment:
Thanks for the bunch of good flags. About to address them all.
For `data_type`, it is needed so that annotations automatically appear on top of images.
The value of interest here is `ANNOTATION/LABEL`.